torch/_export/logical_schema.py

```diff
@@ -292,7 +294,7 @@ class GraphModule:
 
     # Stateful fields of the graph module
 
-    # The name of the tensor will be used to bind to the IValues of Graph Inputs
+    # The name of the tensor will be used to bind to the TensorValues of Graph
     # !!! Consider storing them in the Graph.
     # There are functional difference between buffers and parameters, so they are stored separately.
     parameters: Dict[str, Tensor]
```

Comment:
If node referes to weights which are presumably stored as buffers/parameters, are they also looked up via `TensorArgument`'s `str` param? Right now the comment seems to suggest that `# identifier of the tensor, which must exist in graph's tensor_values`, but tensor_values only capture input, output and intermediate values.